docs/pages/versions/v48.0.0/sdk/image.mdx

```diff
@@ -42,7 +42,7 @@ import { YesIcon, NoIcon, PendingIcon } from '~/ui/components/DocIcons';
 ## Installation
 
 > **Info** Currently `expo-image` can be used only with [development builds](/development/create-development-builds/) and bare React Native apps with [configured Expo modules](/bare/installing-expo-modules/).
-> It is not available in Expo Go and Snack yet.
+> It is available in Expo Go and Snack with Expo SDK 48.
```

Comment:
```suggestion
> It is available in Expo Go with Expo SDK 48 and not available with Snack yet.
```

Hey @flexbox, thanks for updating this! 
It doesn't work with Snack yet since the latest SDK on Snack shown is 47 which will throw the following error for Android and iOS:

<img width="408" alt="CleanShot 2023-03-12 at 01 33 47@2x" src="https://user-images.githubusercontent.com/10234615/224509290-86fe1141-79dd-4019-b371-46cce7f4d4a5.png">

The above uses the snack example you shared as a repro. Let me know if I'm missing something here in case of Snack.